README.md

```diff
@@ -69,7 +69,13 @@
 
 # How to set up a Brave Hub using Ansible
 
-1. follow the generic RPi setup instructions above
+1. use either Balena Etcher or Raspberry Pi Imager to flash the SD card with Raspbian Buster
+
+1. create a file named `ssh` in the boot partition of the SD card
+
+1. install the SD card into the raspberry pi, connect it to power and ethernet
+
+1. plug the RPi into a router that you have admin access to and determine its IP address
 
 1. install ansible (see https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html)
```

Comment:
would it make sense to change this to 'on your local machine, install ansible ...' to clarify that the instructions no longer refer to the RPi environment?

Reply:
Good idea. Done.